site/_includes/contributors.html

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col">
-    <h2 class="mb-2">The {{ site.title }} Team</h2>    
+    <h2 class="mb-2">The {{ site.title }} Team</h2>
```

Comment:
Let's change this to read "Community Members" or something like that.

Reply:
I changed to "Community Members"